examples/tutorials/05_train_sklearn_models.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Remove <code>num_class = dataset.num_classes</code>. Use <code>D.num_classes</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
`range(D.num_classes)` raises error as it executes before named expression is being unwrapped